EpikCord/__init__.py

```diff
@@ -2481,13 +2589,17 @@ def __init__(
         *,
         status: Optional[Status] = None,
         activity: Optional[Activity] = None,
+        cache_limit:Optional[int] = 1000,
```

Comment:
So you did what I said, but didn't properly implement it. You just take in the arg, not use it. And the name is rather vague. Cache limit for what?

Reply:
I did

Reply:
finish changes